src/clusterfuzz/_internal/bot/fuzzers/engine_common.py

```diff
@@ -656,3 +657,34 @@ def get_log_header(command, time_executed):
   """Get the log header."""
   quoted_command = get_command_quoted(command)
   return f'Command: {quoted_command}\nTime ran: {time_executed}\n'
+
+
+def is_sha1_hash(possible_hash):
+  """Returns True if |possible_hash| looks like a valid sha1 hash."""
+  if len(possible_hash) != 40:
+    return False
+
+  hexdigits_set = set(string.hexdigits)
```

Comment:
Can you make this a global constant? I don't like recreating this for every testcase.
Even though it isn't your code.